migration-helper/modelmesh-to-raw.sh

```
@@ -0,0 +1,1445 @@
+        echo -e "${ERROR_SYMBOL} Failed to update secret '$secret_name': $patch_output"
+        ERRORS+=("Failed to update secret '$secret_name' data.URI: $patch_output")
+    fi
+}
+
+process_inference_services() {
+    echo "🔄 Processing InferenceServices for Raw Deployment migration..."
+
+    # First pass: collect all original InferenceServices
+    for isvc_name in "${SELECTED_ISVCS[@]}"; do
+        echo "📋 Collecting original InferenceService '$isvc_name'..."
+        local original_isvc=$(oc get inferenceservice "$isvc_name" -n "$FROM_NS" -o yaml 2>&1)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)